pydocx/export/html.py

```diff
@@ -22,6 +22,30 @@
 
 class PyDocXHTMLExporter(PyDocXExporter):
 
+    def __init__(self, *args, **kwargs):
+        super(PyDocXHTMLExporter, self).__init__(*args, **kwargs)
+        self.heading_level_conversion_map = {
+            'heading 1': 'h1',
+            'heading 2': 'h2',
+            'heading 3': 'h3',
+            'heading 4': 'h4',
+            'heading 5': 'h5',
+            'heading 6': 'h6',
+        }
+        self.default_heading_level = 'h6'
+
+    def load_document(self):
+        document = super(PyDocXHTMLExporter, self).load_document()
```

Comment:
So you're trying to avoid using `self._document`, here? I'm sure I'm missing something, but my reading of this right now is that every reference to `self.document` inside `PyDocXHTMLExporter` will result in a new call to `load_document`, which creates a new `WordprocessingDocument`, which sounds expensive.